.prettierrc

```diff
@@ -1,4 +1,4 @@
 {  
-  "printWidth": 100,  
+  "printWidth": 88,  
```

Comment:
I ended up going with a line length of 88 here as that's what's recommended by Python's Black formatter and it seems nice to have a consistent line length between Python and JS. Generally, keeping line lengths between 60-100 chars is recommended for readability. I'm open to changing this within that range if anyone has strong opinions.

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
Sigh, the 100 was already kind of narrow. Well, I guess we can do it if it's what the rest of the world wants.